packages/appstore/src/components/modals/mt5-migration-modal/mt5-migration-modal.tsx

```diff
@@ -0,0 +1,279 @@
+                                    <Localize i18n_default_text='If you have open positions' />
+                                </Text>
+
+                                <MigrationInfoMessages text_message='Your funds will remain in your existing MT5 account(s).' />
+
+                                <MigrationInfoMessages text_message='You can continue trading on your existing MT5 account(s) until you close all open positions.' />
+
+                                <MigrationInfoMessages text_message='New MT5 account(s) under the BVI jurisdiction will be created for future trades.' />
+                            </div>
+                        </div>
+                        <div className='mt5-migration-modal__existing-accounts-card'>
+                            <div className='mt5-migration-modal__existing-accounts-card-content'>
+                                <Text weight='bold' size={title_size}>
+                                    <Localize i18n_default_text='If you have open positions' />
+                                </Text>
+
+                                <MigrationInfoMessages text_message='Your funds will remain in your existing MT5 account(s).' />
+
+                                <MigrationInfoMessages text_message='You can continue trading on your existing MT5 account(s) until you close all open positions.' />
+
+                                <MigrationInfoMessages text_message='New MT5 account(s) under the BVI jurisdiction will be created for future trades.' />
+                            </div>
+                        </div>
+                    </div>
+                    <div>
+                        <div className=' mt5-migration-modal__existing-accounts-card-content'>
+                            <Checkbox
+                                value={is_checked}
+                                onChange={() => setIsChecked(true)}
+                                label={
+                                    <Text as='p' size={checkbox_text_size} line_height='xs'>
+                                        <Localize
+                                            i18n_default_text='I agree to move my MT5 account(s) and agree to Deriv BVI Ltd’s <0>terms and conditions</0>'
+                                            components={[
+                                                <StaticUrl key={0} className='link' href={'tnc/deriv-(bvi)-ltd.pdf'} />,
+                                            ]}
+                                        />
+                                    </Text>
+                                }
+                                defaultChecked={!!is_checked}
+                            />
+                        </div>
+                    </div>
+                </div>
+            </React.Fragment>
+        );
+    };
+
+    React.useEffect(() => {
+        if (svg_accounts_to_migrate.length) {
+            toggleMT5MigrationModal();
+        }
+    }, []);
```

Comment:
Lets move this useEffect above for better readability and seems like dependency array is missing the properties

Reply:
Changes are handled in parent branch.